src/frontend/components/property-type/phone/edit.tsx

```diff
@@ -27,6 +31,8 @@ const Edit: FC<EditPropertyProps> = (props) => {
           name: property.path,
           required: property.isRequired,
         }}
+        searchPlaceholder={translateLabel('search', resource.id)}
```

Comment:
Let's use `translateMessage` for placeholder too, `translateLabel` is mostly used for translating some headers.

Reply:
Could you also add translations to the core translations for all languages? We usually just use https://www.deepl.com/en/translator for this